src/OrchardCore/OrchardCore.Navigation.Core/NavigationManager.cs

```diff
@@ -203,8 +203,13 @@ private string GetUrl(string menuItemUrl, RouteValueDictionary routeValueDiction
                 menuItemUrl = menuItemUrl.Substring(2);
             }
 
+            if (menuItemUrl.StartsWith("admin/", StringComparison.OrdinalIgnoreCase))
```

Comment:
Need to use admin URL prefix instead, there's a service provide such functionality

Reply:
Thanks @hishamco. I will check and update now